h2d/RenderContext.hx

```diff
@@ -77,6 +77,13 @@ class RenderContext extends h3d.impl.RenderContext {
 	**/
 	@:dox(hide)
 	public var tmpBounds = new h2d.col.Bounds();
+	/**
+		<span class="label">Internal usage</span>
+		
+		Used to calculate filter rendering resolution.
+	**/
+	@:dox(hide)
+	public var tmpPoint = new h2d.col.Point();

```

Comment:
please make it private without documentation and access it with @:privateAccess, it's not meant to be public but implementation detail. Alternatively, you can use a static var in Object.